api/app/src/command/medical/document/document-query.ts

```diff
@@ -133,8 +132,7 @@ export const updateConversionProgress = async ({
     cxId: patient.cxId,
   };
   const { log } = Util.out(`updateConversionProgress - patient ${patient.id}`);
-  let transaction: Transaction | undefined = await startTransaction(PatientModel.prototype);
-  try {
+  return executeOnDBTx(PatientModel.prototype, async transaction => {
```

Comment:
Had to wrap another function on a DB tx, so reckoned we should have it on a shared, reusable function.